src/external/views/nunjucks/content/privacy-policy.njk

```diff
@@ -97,7 +97,7 @@
           national-operator-returns@environment-agency.gov.uk</a>,
         or <span class="govuk-!-font-weight-bold">03708 506 506</span>.
       </p>
-      <p>Lines are open Monday to Friday, 8am to 6pm (GMT).</p>
+      <p>Lines are open Monday to Friday, 8am to 6pm.</p>
```

Comment:
Could we use the contact details macro in the privacy policy pages, or is the content different in some way?